SocialFilter/sections/specific.txt

```diff
@@ -1260,6 +1260,7 @@ lyrical-nonsense.com##.lyricdetailssoc
 utaten.com##.shareArea__item--SNS
 invite.viber.com##div[class^="shareTrigger"]
 pokemongo-master.com##.tweet_box
+pokemongo-get.com##div[class$="cursor"]:has( > i[aria-hidden][class])
```

Comment:
Can you show the screenshot of the file?
Also please send the html file to filters@adguard.com. Those links are dead on my end.

Reply:
> Can you show the screenshot of the file with opened dev tools?

screenshot is this
<details><summary>screenshot</summary>

![](https://i.postimg.cc/k5wfxWRL/image.png)
</details><br/>

> Also please send the html file to [filters@adguard.com](mailto:filters@adguard.com)

I see. I will send e-mail to that adress.

Reply:
Thanks for this, I added the rule directly https://github.com/AdguardTeam/AdguardFilters/commit/969beb5e981389e86c3bcba056213d0ebe63696e.
Note, that such rules should be added to `css_extended.txt`.